infrastructure/clients/pokeapi.go

```diff
@@ -0,0 +1,54 @@
+package pokeapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	models "github.com/hamg26/academy-go-q42021/domain/model"
+)
+
+const apiurl = "https://pokeapi.co/api/v2/"
```

Comment:
Why not set this host in the config file?

Reply:
Good idea! Done